### DISCUSSION TITLE
The Octocat inside the website icon must not be 100% transparent.

### DISCUSSION THREAD

**Discussion by ajaibsingh-at-rb** [community/community#12421] [Other Feature Feedback, Questions, & Ideas]
This is related to the UX.

I am an Ubuntu user and mostly use a dark theme. The tab background in my Chrome is set to black (dark gray).
So when GitHub tab is inactive I am not able to identify that one by looking at the icon. Because:
\+ The browser tab background is BLACK
\+ Github icon is BLACK
\+ The Octocat inside the icon is transparent which also becomes BLACK due to transparency.

So the icon blends with the background.

I am used to keeping many tabs open so the tab size becomes too small as the result title is no more visible.
In that case, the only way to easily identify the GitHub tabs is by looking at the icon (which is not visible in my case).

So I would suggest setting the Octocat in the icon to be white and 50% transparent.
This way the icon will not look different than the current one but will still be visible on black background.

Look at the Image below (the 1st tab):
![image](https://user-images.githubusercontent.com/94602742/156823419-8f5fbea0-b49f-4a7b-94d7-47f3dfae5304.png)

Thank you.

**Comment by mrleblanc101** [comment]
The Github favicon respect dark-mode.
The problem is with your browser or distro.
![Capture d’écran, le 2022-06-29 à 13 21 30](https://user-images.githubusercontent.com/17392251/176497927-3360a9ae-87bf-47e4-932d-f328e8eaad8e.png)
